packages/core/src/makePrepareRealmGlobalFromConfig.js

```diff
@@ -104,17 +104,7 @@ function makePrepareRealmGlobalFromConfig({ createFunctionWrapper }) {
     })
 
     // set circular globalRefs
-    globalThisRefs.forEach((key) => {
-      // if globalRef is actually an endowment, ignore
-      if (topLevelReadAccessKeys.includes(key)) {
-        return
-      }
-      if (topLevelWriteAccessKeys.includes(key)) {
-        return
-      }
-      // set circular ref to global
-      packageCompartmentGlobal[key] = packageCompartmentGlobal
-    })
+    globalThisRefs.forEach((key) => packageCompartmentGlobal[key] = packageCompartmentGlobal)
```

Comment:
I was trying to defend the legacy approach but I'm out of arguments. This is the right thing to do and I don't see a single reason why not (other than some cases might require more work from the user to put stuff in overrides when we don't detect nested fields.
This is a breaking change (even if we could talk around it being one, I want it to be indicated as a breaking change)

Reply:
"that's practically like asking LavaMoat to execute outside of LavaMoat"

Reply:
this is a breaking change in multiple packages, yes?

Reply:
Yes, but only formally. It's the desired behavior and previous was less secure.